entity-command.php

```diff
@@ -32,9 +32,6 @@
 		if ( !is_multisite() ) {
 			WP_CLI::error( 'This is not a multisite installation.' );
 		}
-		if ( \WP_CLI\Utils\wp_version_compare( '5.0', '<' ) ) {
-			WP_CLI::error( "Requires WordPress 5.0 or greater." );
```

Comment:
You'd still need to error out if the WordPress version is not the correct one, i.e. 5.0.